website/app/components/doc/page/hero.hbs

```diff
@@ -1,7 +1,10 @@
 <div class="doc-page-hero" ...attributes>
-  {{#if (has-block)}}
-    <h1 class="doc-text-h1">{{yield}}</h1>
-  {{else}}
-    <h1 class="doc-text-h1">This is an hero block</h1>
-  {{/if}}
+  <div class="doc-page-hero__inner-wrapper">
+    {{#if @title}}
+      <h1 class="doc-page-hero__title doc-text-h1">{{@title}}</h1>
+    {{/if}}
+    {{#if @description}}
+      <p class="doc-page-hero__description doc-text-body-large">{{@description}}</p>
```

Comment:
```suggestion
      <p class="doc-page-hero__description doc-text-body">{{@description}}</p>
```

I must've pulled in the wrong style, but I checked some of the marketing files again and it looks like we want the standard body size, not the large body size. Apologies for the oversight; I've updated the figma wires already.